eslint-plugin-relay.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     'generated-flow-types': require('./src/rule-generated-flow-types'),
     'no-future-added-value': require('./src/rule-no-future-added-value'),
     'unused-fields': require('./src/rule-unused-fields'),
+    'unused-fragments': require('./src/rule-unused-fragments'),
```

Comment:
Internally, we call this rule `must-colocate-fragment-spreads`. We do this because this lint rule catches:

1. Situations where a fragment is spread in module A, but the module (B) that defines that fragment, is not imported by module A. This is an anti-pattern we want to lint against.
2. As a big side benefit, this lint rule will *also* detect if a fragment is possibly unused

So the current name for the rule should be updated to reflect that.

Reply:
@poteto I was thinking, how are you handling mutations with this rule? I found it to be quite common to have mutations fetching some extra data via fragment spread but the module is actually not directly imported near the mutation. Example: `addNote` mutation which uses fragment from some `Note` component so you can append it to the list. 🤔

Reply:
This rule ignores mutations and subscriptions.